src/python/install.sh

```diff
@@ -300,15 +307,6 @@ install_using_oryx() {
     add_symlink
 }
 
-sudo_if() {
-    COMMAND="$*"
-    if [ "$(id -u)" -eq 0 ] && [ "$USERNAME" != "root" ]; then
-        su - "$USERNAME" -c "$COMMAND"
-    else
-        $COMMAND
-    fi
-}
-
 install_user_package() {
```

Comment:
As `install_user_package` is no longer used, can we clean this?

Reply:
Done in 5ef0e17ad864f14cbc83406701a2b8ab64b4aebf